super/pom.xml

```diff
@@ -92,7 +92,7 @@
     <maven.eclipse.plugin.version>2.10</maven.eclipse.plugin.version>
     <maven.exec.plugin.version>1.6.0</maven.exec.plugin.version>
     <jaxb2.maven.plugin.version>2.5.0</jaxb2.maven.plugin.version>
-    <spotbugs.maven.plugin.version>3.1.12.2</spotbugs.maven.plugin.version>
+    <spotbugs.maven.plugin.version>4.0.0</spotbugs.maven.plugin.version>
```

Comment:
The maven plugin configuration in pluginManagement for the SpotBugs maven plugin has a dependency override for its transitive dependency to SpotBugs itself. Maybe the explicit override is not needed anymore.
I wonder if the dependabot notifies if the transitive dependency gets outdated. In that case it is probably OK to leave the configuration in pluginManagement as it is.